documentation/bolt_installing.md

```diff
@@ -320,6 +320,11 @@ Puppet Tools on Linux 7 is `puppet-tools-release-el-7.noarch.rpm`.
         sudo rpm -Uvh https://yum.puppet.com/puppet-tools-release-fedora-31.noarch.rpm
         sudo dnf install puppet-bolt
         ```
+    -   Fedora 31
```

Comment:
```suggestion
    -   Fedora 32
```

Reply:
I have no idea how that happened I SWEAR